Lib/test/test_buffer.py

```diff
@@ -4439,6 +4439,13 @@ def test_issue_7385(self):
         x = ndarray([1,2,3], shape=[3], flags=ND_GETBUF_FAIL)
         self.assertRaises(BufferError, memoryview, x)
 
+    def test_bytearray_release_buffer_read_flag(self):
+        # See https://github.com/python/cpython/issues/126980
+        with self.assertRaises(SystemError):
+            bytearray().__buffer__(inspect.BufferFlags.READ)
+        with self.assertRaises(SystemError):
+            bytearray().__buffer__(inspect.BufferFlags.WRITE)
```

Comment:
Maybe use an non-empty bytearray:

```suggestion
        # See https://github.com/python/cpython/issues/126980
        obj = bytearray(b'abc')
        with self.assertRaises(SystemError):
            obj.__buffer__(inspect.BufferFlags.READ)
        with self.assertRaises(SystemError):
            obj.__buffer__(inspect.BufferFlags.WRITE)
```

Reply:
Ok!